rust/src/ftp/constant.rs

```diff
@@ -0,0 +1,61 @@
+// FTP state progress values
+pub const FTP_STATE_NONE: u8 = 0;
+pub const FTP_STATE_IN_PROGRESS: u8 = 1;
+pub const FTP_STATE_PORT_DONE: u8 = 2;
+pub const FTP_STATE_FINISHED: u8 = 3;
+
+// FTP Data progress values
+pub const FTPDATA_STATE_IN_PROGRESS: u8 = 1;
+pub const FTPDATA_STATE_FINISHED: u8 = 2;
+
+// FTP request command values
+pub const FTP_COMMAND_UNKNOWN: u8 = 0;
```

Comment:
Should this be an enum ?

Reply:
Is there an advantage to using an enum over a constant?

Reply:
switch cases are more elegantly handled by the rust compiler...

And this looks more readable to me... to group these constants together